src/libraries/icubmod/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ yarp_begin_plugin_library(icubmod QUIET)
   add_subdirectory(embObjAnalog)
   add_subdirectory(embObjStrain)
   add_subdirectory(embObjFTsensor)
+  add_subdirectory(embObjMultipleFTsensor)
```

Comment:
We agreed to call it `embObjMultipleFTsensors` with ending `s`.
Was it done on purpose? If so, is there any reason behind it that I don't catch now?
Maybe, was it because the name of a device is singular?

Adding the trailing `s` would entail other updates here and there, but better now than later.

Reply:
Ok, I will do it at the end as the last thing. Otherway I fear misaligning all the source code for review.

Reply:
Good point 👍🏻

Reply:
Let me keep this unresolved otherwise I'm gonna forget it 😄

Reply:
Done via https://github.com/robotology/icub-main/pull/786/commits/d8335023aefc33fadc81b3a2903ae776e890fd08.
Thanks @triccyx 👍🏻